packages/kbn-monaco/src/console/output_parser.js

```diff
@@ -0,0 +1,387 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/* eslint-disable prettier/prettier,prefer-const,no-throw-literal,camelcase,@typescript-eslint/no-shadow,one-var,object-shorthand,eqeqeq */
+
+export const createOutputParser = () => {
```

Comment:
This is a very similar version to the request console parser, but modified to only support selecting json payloads with a `#` comment right above them. I decided to just dupe most of the code since it was quite complex to just modify the existing one to accomodate both cases (requests as input and results as output)